src/algorithm.jl

```diff
@@ -112,11 +112,33 @@ algorithm `algo` with verbose level `verbose`.
 """
 function sample_scenarios end
 
+function sample_scenarios(sp::AbstractStochasticProgram, n::Int, depth_max = 1000,to::TimerOutput=TimerOutput(), verbose::Int=0)
```

Comment:
80C. 
For consistency, we should keep the PEP8 all along : 
- add space after comma: `, to::...`
- no space before and after equal symbol "=" in function arguments.